packages/react-cosmos/src/components/proxies/PropsProxy.jsx

```diff
@@ -10,11 +10,15 @@ export default class PropsProxy extends React.Component {
       fixture,
       onComponentRef,
     } = this.props;
+    let children;
+    if (Array.isArray(fixture.children)) {
+      children = [].slice.call(fixture.children);
+    }
 
     return React.createElement(component, {
       ...fixture,
```

Comment:
We need to omit the `children` field from the fixture, otherwise it continues to be sent as a prop as well as the 3rd arg from `createElement`.  See [how StateProxy omits `state` field](https://github.com/react-cosmos/react-cosmos/blob/b3be0680336724bce8830dae2f367df260394364/packages/react-cosmos/src/components/proxies/StateProxy.jsx#L100).

You also need to add a test that checks that `fixture.children` is no longer sent as a prop.